src/utils/base64.ts

```diff
@@ -1,15 +1,19 @@
 export { textToBase64, base64ToText, isValidBase64, removePotentialDataAndMimePrefix };
 
-function textToBase64(str: string) {
-  return window.btoa(str);
+function textToBase64(str: string, urlSafe = false) {
+  const encoded = window.btoa(str);
+  return urlSafe ? makeUriSafe(encoded) : encoded;
 }
 
-function base64ToText(str: string) {
-  if (!isValidBase64(str)) {
+function base64ToText(str: string, urlSafe = false) {
```

Comment:
I'm not really fan of magic boolean positionnal arguments, can you make this argument a config object please ?
```suggestion
function base64ToText(str: string, { makeUrlSafe = false } : { makeUrlSafe?: boolean } = {}) {
```